app/controllers/structures_controller.rb

```diff
@@ -9,14 +9,14 @@ def show
     respond_to do |format|
       format.csv do
         # Download the structural spreadsheet
-        cocina = Repository.find(params[:item_id])
+        cocina = fetch_cocina(item_id: params[:item_id])
```

Comment:
Maybe this goes beyond the scope of the issue you're working on and verges into a different one, but I'm wondering if this call should also have `, user_version_id: params[:user_version_id]`

Reply:
Yeah, probably beyond the scope since this change will also need a path helper change to use the user_version_aware path?

Reply:
@justinlittman do we need a ticket for ☝🏻 or is this not in the requirements? "This" being user version-aware structure CSV downloads.

Reply:
I'm not sure I'm following. We already add version-aware routes for other endpoints. Is this what you mean? https://github.com/sul-dlss/argo/blob/main/config/routes.rb#L221-L232

Reply:
I think it probably should, and was the crux of the original DRYing, so adding it is a good catch.